crates/rrg/src/args.rs

```diff
@@ -49,6 +49,14 @@ pub struct Args {
            arg_name="PATH",
            description="whether to log to a file")]
     pub log_to_file: Option<std::path::PathBuf>,
+
+    /// The public key for verfying signed commands.
+    #[argh(option,
+       long="command-verification-key",
+       arg_name="KEY",
+       description="verification key for signed commands",
```

Comment:
Let's specify that it expects hex encoded bytes, e.g. _"verification key for signed commands (hex-encoded bytes)"_.

Reply:
Done!